nucypher/policy/collections.py

```diff
@@ -279,6 +279,7 @@ def __len__(self):
 
     @classmethod
     def construct_by_bob(cls, arrangement_id, capsules, ursula, bob):
+        ursula.mature()
```

Comment:
Stuff like this makes me think that Bob is a condescending jerk.

Reply:
He has no interest in an immature Ursula.  He's sophisticated.  Classy.  And he's got a long adventure ahead of him.  This is no place for a `NodeSprout`.